src/utils.tsx

```diff
@@ -0,0 +1,21 @@
+import _ from 'lodash';
+
+function makeid(length: number) {
+  let result = '';
+  let characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+  let charactersLength = characters.length;
+  for (let i = 0; i < length; i++) {
+    result += characters.charAt(Math.floor(Math.random() * charactersLength));
+  }
+  return result;
+}
+
+function getCurrentSection(id: string): string {
+  return id.split('_')[1];
+}
+
+function getCurrentField(id: string): string {
+  return _.last(id.split('_')) || '';
```

Comment:
You should add comment to explain the structure of the `id` to help reader to understand why you split this string and retrieve value at specific index.